tests/src/Interop/COM/ServerContracts/Server.Contracts.cs

```diff
@@ -146,7 +146,7 @@ public interface IStringTesting
 
         void Reverse_LPWStr_Out([MarshalAs(UnmanagedType.LPWStr)] string a, [MarshalAs(UnmanagedType.LPWStr)] out string b);
 
-        void Reverse_LPWStr_OutAttr([MarshalAs(UnmanagedType.LPWStr)] string a, [Out][MarshalAs(UnmanagedType.LPWStr)] string b);
+        void Reverse_LPWStr_OutAttr([MarshalAs(UnmanagedType.LPWStr)] string a, [MarshalAs(UnmanagedType.LPWStr)] string b);
```

Comment:
The name is wrong now. Should this entirely be removed? I was under the impression we should ensure this scenario properly gets an exception thrown.

Reply:
Also the other native contract now need to updated.

Reply:
Yeah I'll change this to check for the exception.